### PR TITLE
Set timezone to UTC for all storedprocedure timestamps

### PR DESCRIPTION
**Pull request by lmsurpre**
also added `shutdownDatabase` placeholder in AbstractPersistenceTest

Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractPersistenceTest.java [hunk 1]
```diff
@@ -97,6 +101,11 @@ public void commitTrx() throws Exception{
             persistence.getTransaction().commit();
         }
     }
+    
+    @AfterClass(alwaysRun = true)
```
Comment (prb112) [inline fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractPersistenceTest.java]: Look at AfterSuite

Comment (lmsurpre) [reply fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractPersistenceTest.java]: is it ok to have the same AfterSuite method on every test class?

File: fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchDateTest.java [hunk 2]
```diff
@@ -74,7 +70,7 @@ public void testSearchDate_date() throws Exception {
         assertSearchReturnsSavedResource("date", "lt2018-10-29T17:12:00-04:00");
         assertSearchReturnsSavedResource("date", "gt2018-10-29T17:12:00-04:00");
         assertSearchReturnsSavedResource("date", "le2018-10-29T17:12:00-04:00");
-        
+        assertSearchReturnsSavedResource("date", "gt2018-10-29T17:12:00-04:00");
```
Comment (lmsurpre) [inline fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchDateTest.java]: duplicate of two lines above

File: fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java [hunk 3]
```diff
@@ -651,6 +651,8 @@ private void setDateValues(DateParmVal p, Date date) {
             p.setValueDateStart(DateTimeHandler.generateTimestamp(inst));
             inst = DateTimeHandler.generateUpperBound(date.getValue());
             p.setValueDateEnd(DateTimeHandler.generateTimestamp(inst));
+            
+            System.out.println(p);
```
Comment (lmsurpre) [inline fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java]: note to self:  make sure we remove this before merging

Comment (prb112) [reply fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java]: should be removed

File: fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchDateTest.java (hunk 2 above)
Comment (prb112) [reply fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchDateTest.java]: fixed

**Pull request merged**